src/server/prod.ts

```diff
@@ -6,21 +6,53 @@ import minimist from 'minimist';
 import favicon from 'serve-favicon';
 import sio from 'socket.io';
 
-import { WebSocketProxyUDPServer } from './network/network';
-import { WebSocketServer } from './network/web_socket_server';
+import { UDPServers } from './network/network';
 
+// Process command line arguments
 const args = minimist(process.argv.slice(2));
-const teamAAddress = args.team_a || '239.226.152.162';
-const teamBAddress = args.team_b || '239.226.152.163';
+
+// Get all of the teams from the command line
+// Fall back to our defaults if none were specified
+let teams = args._;
+if (teams.length === 0) {
+  teams = ['Red Team:239.226.152.162:9917', 'Blue Team:239.226.152.163:9917'];
+}
+
+// Convert command line data into a more usable format
+const teamData = teams.map((team: string) => {
+  const nameSplit = team.indexOf(':');
```

Comment:
```suggestion
  const [name, address, port]= team.split(':');
  return { name, address, port: Number(port) };
```

Reply:
We are allowing for IPv6 addresses which likely contain `:` so we need to specifically look for the first and last occurrences, or choose a different separator

Reply:
Ah. Overlooked that. Perhaps `|` as a separator? Unlikely to be in a team name.

Reply:
`/` is also an option that I thought of

Reply:
`:` was chosen because it started with IPv4 and thats common for `<address>:<port>` notation

Reply:
The other option is we let the shell separate the tokens and use spaces, every 3 values is a name, IP, port

Reply:
Except its not uncommon for team names to have spaces

Reply:
That's what quotes are for

Reply:
Thats a point, they would need quotes no matter what separator we use

Reply:
But you would no longer have the confusion between the name, IP and port. Because I imagine you could in theory have colons in the team name too